src/shared/components/setup-locker/setup-locker-steps/set-timeout/SetTimeout.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import { IdlePicker, Button } from '@nomios/web-uikit';
+import PropTypes from 'prop-types';
+
+import styles from './SetTimeout.css';
+
+class SetTimeout extends Component {
+    state = {
+        timeoutValue: 3,
+    };
+
+    render() {
+        const { timeoutValue, feedback } = this.state;
+
+        return (
+            <div className={ styles.contentWrapper }>
+                <h2 className={ styles.title }>Setup expiration time.</h2>
+                <p>You may tweak the amount of time you are ablo to use
+                 the app before it locks again. You can go have a coffee without any worries.</p>
+                <div className={ styles.idlePicker }>
+                    <IdlePicker defaultValue={ timeoutValue } handlePickerChange={ this.onPickerChange } />
```

Comment:
`handlePickerChange` was changed to `onChange` in IdlePicker.

Also the callback should be named `handlePickerChange` and no `onPickerChange`. For reference, callback props always start with `onXXX` and the handlers with `handleXXXX`.